m4_sessions/1901-inning/social-movements.md

```diff
@@ -74,3 +74,19 @@ _We'll split the room into 5 groups and each of the presenters will sit with a g
 
 **4. Share out:** (10 minutes)
 - What is your personal committment (post Turing) to benefit the greater good as a developer (i.e. with your new skills)? 
+
+
+
+
+
+_____________________________________________________________________
+
+Our plan for changes: 
+
+Altogether, we like the set up of this lesson and want to stick to the format of it, however we do want to make some changes. 
+
+
+Things to be changed:
+Isaac - Come up with more relevant examples for social movements in tech. He will use the four types of social movements(Alternative, Redemptive, Revolutionary, Reformative).
```

Comment:
This is awesome!  Issac, please make sure to post resources in the markdown and in the channel